scripts/kind.yaml

```diff
@@ -1,6 +1,6 @@
 kind: Cluster
 apiVersion: kind.sigs.k8s.io/v1alpha3
 nodes:
-- role: control-plane
```

Comment:
Do we need this changes here?

Reply:
oops, it's a typo. I'll fix in next commit, thanks